etc/python-gdeploy.conf

```diff
@@ -0,0 +1,2 @@
+[python-gdeploy]
+glusterfs_repo = https://buildlogs.centos.org/centos/7/storage/x86_64/gluster-3.9/
```

Comment:
okay, will make this a sample file and ask users to copy it and modify accordingly if needed